client/src/components/files/FileMoveItem.vue

```diff
@@ -18,79 +18,70 @@
-          </span>
-          <span
-            class="hover-state"
-            v-show="[FileOperationState.EntryMoved].includes(state)"
-          >
-            {{ $msTranslate('FoldersPage.MoveFile.browse') }}
-          </span>
-        </ion-label>
-      </div>
-
-      <!-- waiting -->
-      <div class="waiting-info">
-        <ion-text
-          class="waiting-text body"
-          v-if="state === FileOperationState.MoveAdded"
+        <ion-label
+          class="element-details__workspace body-sm"
+          v-if="workspaceInfo"
         >
-          {{ $msTranslate('FoldersPage.MoveFile.waiting') }}
-        </ion-text>
-        <ion-button
-          fill="clear"
-          size="small"
-          class="cancel-button"
-          v-if="state === FileOperationState.MoveAdded"
-          @click="$emit('cancel', operationData.id)"
+          {{ workspaceInfo.currentName }}
+        </ion-label>
+        <div
+          class="element-details-progress-container"
+          v-show="state === FileOperationState.OperationProgress"
         >
-          <ion-icon
-            class="cancel-button__icon"
-            slot="icon-only"
-            :icon="close"
+          <ion-progress-bar
+            class="progress-bar"
+            :value="getProgress() / 100"
           />
-        </ion-button>
+          <ion-button
+            fill="clear"
+            size="small"
+          >
+            <ion-icon
+              class="cancel-icon"
+              v-if="state === FileOperationState.CopyAdded"
+              @click="$emit('cancel', operationData.id)"
+              type="button"
+              :icon="closeCircle"
+              slot="icon-only"
+            />
+          </ion-button>
+        </div>
       </div>
 
-      <!-- in progress -->
-      <ion-button
-        fill="clear"
-        size="small"
-        class="cancel-button"
-        v-if="state === FileOperationState.OperationProgress"
-        @click="$emit('cancel', operationData.id)"
+      <!-- waiting -->
+      <div
+        class="waiting-info"
+        v-if="state === FileOperationState.MoveAdded"
       >
+        <ion-text class="waiting-text body">
+          {{ $msTranslate('FoldersPage.MoveFile.waiting') }}
+        </ion-text>
         <ion-icon
-          class="cancel-button__icon"
+          class="cancel-icon"
+          @click="$emit('cancel', operationData.id)"
+          type="button"
+          :icon="closeCircle"
           slot="icon-only"
-          :icon="close"
         />
-      </ion-button>
+      </div>
 
       <!-- done -->
       <ion-icon
-        class="checkmark-icon default-state"
+        class="checkmark-icon"
         v-show="state === FileOperationState.EntryMoved"
-        :icon="checkmark"
-      />
-      <ion-icon
-        class="arrow-icon hover-state"
-        v-show="state === FileOperationState.EntryMoved"
-        :icon="arrowForward"
+        @click="$emit('cancel', operationData.id)"
```

Comment:
Why `@click=cancel()`? The entry has been moved, you don't want to (and can't) cancel the move now.